ReClass.NET/Forms/PluginForm.Designer.cs

```diff
@@ -115,7 +115,7 @@ private void InitializeComponent()
 			this.pluginsDataGridView.SelectionMode = System.Windows.Forms.DataGridViewSelectionMode.FullRowSelect;
 			this.pluginsDataGridView.Size = new System.Drawing.Size(702, 197);
 			this.pluginsDataGridView.TabIndex = 0;
-			this.pluginsDataGridView.SelectionChanged += new System.EventHandler(this.pluginsDataGridView_SelectionChanged);
+			this.pluginsDataGridView.SelectionChanged += new System.EventHandler(this.PluginsDataGridView_SelectionChanged);
```

Comment:
You made this method name upper case but the real method name is lowercase and should stay lower case like every other autogenerated event handler.

Reply:
i honestly have no idea how this changed lol

Reply:
turns out i had changed it, but i reverted it and it didn't reflect in the designer